OSDOCS-16988-4-CQA-1:CQA PR 4.1 for INSTALL-1: Core Installation Conc… (#113598)
openshift/openshift-docs: modules/reviewing-the-installation-log.adoc

[id="reviewing-the-installation-log_{context}"]
= Reviewing the installation log

[role="_abstract"]

> The {product-title} installation log contains a summary of the installation. If installation succeeds, the log includes the information required to access the cluster.

Reviewer: I know this is existing text, but _If installation succeeds_ seems weak to me. Of course the install will succeed!! Maybe:

```suggestion
The {product-title} installation log contains a summary of the installation, including the information required to access the cluster after installation.
```

Author: :heavy_check_mark:
